package.json

```diff
@@ -17,8 +17,8 @@
     "lint:js": "eslint . --ext .js,.ts,.tsx",
     "lint:css": "stylelint src",
     "test": "npm run test:unit:ci",
-    "test:unit": "ELECTRON_RUN_AS_NODE=true electron $(which jest)",
-    "test:unit:ci": "ELECTRON_RUN_AS_NODE=true electron $(which jest) --ci --collectCoverage"
+    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=true electron node_modules/.bin/jest-runtime",
```

Comment:
Does this work? If not, let's not keep it.

Reply:
so remove the lines completely? or just the changes I made when trying to fix it?